torch_geometric/nn/conv/message_passing.py

```diff
@@ -184,6 +184,34 @@ def __init__(
         self._edge_update_forward_pre_hooks = OrderedDict()
         self._edge_update_forward_hooks = OrderedDict()
 
+        # Init NVTX Ranges for this op
+        if os.environ.get('NVIDIA_NVTX_RANGES',
+                          "0") == "1" and torch.cuda.is_available():
+            self._nvtx_handles = dict()
+
+            def get_hooks_for(func_name):
```

Comment:
I think this is not an issue if `NVIDIA_NVTX_RANGES` is not meant to be used by the community, but I prefer to define the hook at the top-level to avoid `PicklingError` in a multiprocessing setting. https://docs.python.org/3/library/pickle.html#what-can-be-pickled-and-unpickled